authors/apps/authentication/serializers.py

```diff
@@ -135,12 +135,15 @@ class UserSerializer(serializers.ModelSerializer):
     password = serializers.CharField(
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True,
+        required=True,
+        style={'input_type': 'password', 'placeholder': 'Password'
+               }
     )
 
     class Meta:
         model = User
-        fields = ('email', 'username', 'password')
+        fields = ('password',)
```

Comment:
@ManuelDominic , could you please provide a reason why you are removing the `email` and `username` fields here